### PR TITLE
[T4A4][W13-B1] Lim Hui Qi

### PR DESCRIPTION
**Pull request by LuMiN0uSaRc**
Use Interfaces to apply ISP

### CODE REVIEW

**Comment by LuMiN0uSaRc** [comment]
Ready to review

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/CommandResult.java:14]: this commit is unrelated to the activity here.
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.

File: src/seedu/addressbook/data/person/Printable.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.data.person;
+
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:2]: Please include a header comment to indicate what this interface/class/method does?

File: src/seedu/addressbook/data/person/Printer.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+ 
+import java.util.StringJoiner;
+ 
+public class Printer {
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printer.java:5]: :+1: cool! you created a class that provides this method! 

but, you missed the header comment for the method or the class ;-)

**Comment by okkhoy** [comment]
@LuMiN0uSaRc 
Some comments added. Please ack & close the PR after reading comments.

**Comment by LuMiN0uSaRc** [comment]
Noted with thanks!

**Pull request closed without merging**